.pipeline/scripts/get-maven-excludes.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+import argparse
+
+def write_maven_excludes_github_output(filter_key, filter_value):
+    with open("module-inventory.json", "r") as file:
+        module_inventory = json.load(file)
+
+        filtered_inventory = filter(lambda x: x[filter_key] == filter_value, module_inventory)
+        artifact_ids = list(map(lambda x: x["artifactId"], filtered_inventory))
+        artifact_ids_for_maven_excludes = list(map(lambda x: "!:" + x, artifact_ids))
+
+        excludes = ','.join(artifact_ids)
+        print("Excludes: ",excludes)
```

Comment:
[minor - code style]

```suggestion
        print("Excludes: ", excludes)
```